src/parser/shaman/enhancement/modules/Abilities.tsx

```diff
@@ -1,26 +1,40 @@
 import SPELLS from 'common/SPELLS';
 
 import CoreAbilities from 'parser/core/modules/Abilities';
+import { SpellbookAbility } from 'parser/core/modules/Ability';
+import SpellLink from 'common/SpellLink';
+import React from 'react';
+import { STORMSTRIKE_CAST_SPELLS } from '../constants';
 
 class Abilities extends CoreAbilities {
-  spellbook() {
+  spellbook(): SpellbookAbility[] {
     const combatant = this.selectedCombatant;
     return [
       {
         spell: SPELLS.ASCENDANCE_TALENT_ENHANCEMENT,
         category: Abilities.SPELL_CATEGORIES.COOLDOWNS,
         cooldown: 180,
+        gcd: {
+          base: 1500,
+        },
         enabled: combatant.hasTalent(SPELLS.ASCENDANCE_TALENT_ENHANCEMENT.id),
         castEfficiency: {
           suggestion: true,
           recommendedEfficiency: 1.0,
         },
       },
       {
-        spell: SPELLS.EARTHEN_SPIKE_TALENT,
+        spell: SPELLS.FERAL_SPIRIT,
+        buffSpellId: [ //Feral Spirit isn't an actual buff, so we can only show the Elemental
+          // Spirits buffs
+          SPELLS.ELEMENTAL_SPIRITS_BUFF_MOLTEN_WEAPON.id,
+          SPELLS.ELEMENTAL_SPIRITS_BUFF_ICY_EDGE.id,
+          SPELLS.ELEMENTAL_SPIRITS_BUFF_CRACKLING_SURGE.id,
+        ],
         category: Abilities.SPELL_CATEGORIES.COOLDOWNS,
-        enabled: combatant.hasTalent(SPELLS.EARTHEN_SPIKE_TALENT.id),
-        cooldown: 20,
+        cooldown: combatant.hasTalent(SPELLS.ELEMENTAL_SPIRITS_TALENT)
+          ? 90
+          : 120,
```

Comment:
I do not mind changing it to single line, however according to the code style defined in the the /.idea/codeStyles/Project.xml ide project file. The character margin per lines is supposed to be a maximum of 80 characters for both JavaScript and TypeScript. Markdown is limited to 72 characters and every other file format has a limit of 999 charactes. Please let me know how you wish me to handle this because my IDE will always listen to the styles defined in that file.

Reply:
After working on my other PR for balance druid, it has come to my attention that i'm not the only one making use of this wrapping. I'm receiving mixed singles because of that.
https://github.com/WoWAnalyzer/WoWAnalyzer/blob/master/src/parser/hunter/beastmastery/modules/Abilities.tsx#L68

Reply:
tbf I have a mind of making them all single line in my shadowlands pushes.

Reply:
Also being able to find a few modules where it is split doesnt mean you should be splitting it. like 95%+ of the modules are not splitting things into several lines.

Reply:
Should be fixed throughout enhancement shaman